core/tests/test_apps.monero.bulletproof.py

```diff
@@ -285,102 +285,27 @@ def test_masks(self):
         bpi.use_det_masks = False
         self.mask_consistency_check(bpi)
 
-    def test_verify_testnet(self):
```

Comment:
Why is this test removed?

Reply:
This type of proofs is no longer supported. See `verify_testnet` in `bulletproof.py ` and the commit message: 
> proof_v8 support discontinued, old hardfork, not needed anymore

I think it has something to do with [the old vulnerability](https://www.getmonero.org/2017/05/17/disclosure-of-a-major-bug-in-cryptonote-based-currencies.html).